source/server/config/network/mongo_proxy.cc

```diff
@@ -5,28 +5,32 @@
 #include "envoy/network/connection.h"
 #include "envoy/registry/registry.h"
 
-#include "common/json/config_schemas.h"
+#include "common/config/filter_json.h"
+#include "common/json/json_loader.h"
```

Comment:
nit: do we need this include now? (I think you just need interface JSON include). If so please fix BUILD file also.